inspirehep/modules/workflows/tasks/arxiv.py

```diff
@@ -264,6 +263,10 @@ def _author_list(obj, eng):
                     extracted_authors.extend(marcxml2record(authors_xml).get('authors', []))
 
             if extracted_authors:
+
```

Comment:
I can't think of a reason to separate an `if` from its block: they logically go together, so they should be together in the code. So:
```python
if extracted_authors:
    for author in extracted_authors:
        author['full_name'] = decode_latex(author['full_name'])
    obj.data['authors'] = extracted_authors
```